doc/user/SUPPORTED_FORMATS.md

```diff
@@ -6,14 +6,15 @@ F3D supports the following file formats:
 | ----------------------------------------- | ---------------------------------------------- | ---------- | -------------------- | --------- |
 | Legacy VTK                                | `.vtk`                                         | No         | No                   | `native`  |
 | VTK XML                                   | `.vtp`, `.vtu`, `.vtr`, `.vti`, `.vts`, `.vtm` | No         | No                   | `native`  |
+| VTKHDF                                    | `.vtkhdf`                                      | No         | No                   | `hdf`     |
```

Comment:
I'd assume animated vtkhdf should work ?

Reply:
yes, fixed